x/pairing/types/relay_exchange.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+import (
+	"bytes"
+	"encoding/binary"
+	"strings"
+
+	"github.com/lavanet/lava/utils"
+)
+
+type RelayExchange struct {
```

Comment:
doc: what is this for and how it is used.

Reply:
done. see https://github.com/lavanet/lava/commit/289c1691a0b9a26c10177ed57a5f774e5a54dafb